src/arcade/core/util/Plane.java

```diff
@@ -0,0 +1,82 @@
+package arcade.core.util;
```

Comment:
After reading this, I think it makes more sense to put this under `potts.env.location` given that it is only used for the split function and I can't think of a reason it would be used in `patch`. Then, you can swap the `point` type to `Voxel` so you won't need all the `new Int3D(voxel.x, voxel.y, voxel.z)` calls. Thoughts?